src/EnergyPlus/Photovoltaics.cc

```diff
@@ -1497,8 +1497,8 @@ namespace Photovoltaics {
                 Real64 &XX,
                 std::function<Real64(EnergyPlusData &state, Real64 const, Real64 const, Real64 const, Real64 const, Real64 const, Real64 const)> FXX,
                 std::function<Real64(EnergyPlusData &state, Real64 const, Real64 const, Real64 const, Real64 const, Real64 const)> DER,
-                Real64 const II, // Autodesk Aliased to XX in some calls
-                Real64 const VV, // Autodesk Aliased to XX in some calls
+                Real64 const &II, // Autodesk Aliased to XX in some calls
+                Real64 const &VV, // Autodesk Aliased to XX in some calls
```

Comment:
Debugger showed an uninitialized argument in the call to the NEWTON function for argument II. Changed the argument declarations back to what it was previously corrected the example file failures for GeneratorswithPV and ShopWithBIPVT.